docker/mock_test_handler.py

```diff
@@ -0,0 +1,44 @@
+"""Mock Data Handler for HTTP Request Testing"""
+import http.server
+import json
+
+
+class MockDataHandler(http.server.BaseHTTPRequestHandler):
+  """Mock data handler for testing."""
+  last_modified = 'Tue, 13 Jun 2023 00:00:00 GMT'
+  file_path = 'testdata/rest_test.json'
+
+  def do_GET(self):  # pylint: disable=invalid-name
+    """Serve a mock GET request."""
+    try:
+      with open(self.file_path, 'r') as f:
+        data = json.load(f)
+      if self.path != '/':
+        found = False
+        # find the vulnerability to mock serve the new page
+        for vuln in data:
+          if vuln['id'] == (self.path.split('/')[-1]).split('.json')[0]:
```

Comment:
`vuln['id']` has the prefix of `CURL` but the file name in link starts with `CVE`?